### ISSUE TITLE
Can Not Generate "Initial Code"

### ISSUE THREAD

**Issue by in-a-days** [sakirtemel/MMM-alexa#5]
Hello, I'm having a hard time with the installation.  I have an AVS account set up on Amazon, but I can't seem to figure out how to generate the Initial Code for my config.js.  I have tried this link

https://sakirtemel.github.io/MMM-alexa/

but when I use the information provided by Amazon and click "Request Code" it takes me to an Amazon page where I receive the following error:

"We're sorry!
An error occurred when we tried to process your request. Rest assured, we're already working on the problem and expect to resolve it shortly."

Any help will be appreciated!

**Comment by in-a-days** [comment]
It was my fault!  Did not properly define my whitelist on Amazon AVS.

**Comment by lucallmon** [comment]
This happened to me too. What do you mean, "did not properly define my whitelist?"

**Comment by in-a-days** [comment]
I'm actually running the AlexaPi build, so I don't know if this will exactly solve your problem.  But I got a similar error making the same mistake with both programs.

In your AVS Control Panel, go to your device and check out the Security Profile.  Click on the second tab over, Web Settings.  In order to validate I had to add the local address of my PI device - 192.168.0.xxx (your # will vary) to the "Allowed Origins."  I also had to add 192.168.0.xxx/code to "Allowed Return URLs."  Then when you proceed to the Authentication make sure you run the code generator from the desktop of the PI itself, rather than another system.

I actually had to include port 5050 in my Allowed Origin/URL so it looks like 192.168.0.117:5050 and 192.168.0.117:5050/code respectively.  But again - I used the AlexaPi installer to get my system working so I don't know exactly the configuration that will make this activator work for you.
